### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Properties_Toolbar_NonTextContrast: In System Default/Light theme Color Contrast of focus border on "TestBlob" and other elements in toolbar is less than 3:1

### ISSUE THREAD

**Issue by Jtith** [microsoft/AzureStorageExplorer#6371]
https://github.com/microsoft/AzureStorageExplorer/issues/6371#tasklist-block-aee340e3-b122-4f9a-b306-f4f0614278c3

**GitHub Tags:**

#A11yMAS;#A11yTCS;#Mac;#DesktopApp;#A11ySev2;#BM_AzureToolsStorageExplorer_Mac_Dec2022;#Azure Tools Storage Explorer;#WCAG1.4.11;#A11yWCAG2.1;#ColorContrast;#NonTextContrast;#AINotInScope; 
## Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.27.0
## OS Details
Mac version: macOS Ventura 13.1

## Repro Steps:

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign into Azure.
3. Go to "Manage Accounts" pane and select the subscriptions you will use. Once selected, click on "Apply" button.
4. Select "Blob Containers" of the selected subscription.
5. Navigate to any folder under "Blob" and hit enter and then tab till toolbar menu
6. Use Color Contrast Tool to check the color contrast of keyboard focus border on toolbar section is less than 3:1 or not.

## Actual results:
In System Default/Light theme Color Contrast of focus border on "TestBlob" and other elements in toolbar is less than 3:1 

**Similar issues observed in the following:**

- In System Default/Light theme,

1. Click on any Blob container, Toolbar menu is visible. All the tools present in the above table is failing color contrast for visible focus.
2. Click on any Disk, Toolbar menu is visible. All the tools present in the above table is failing color contrast for visible focus.
3. Click on any Blob container. Tab to properties present at the left bottom is failing color contrast of properties border is less than 3:1
4. Click on any blob container and right to select Get shared access signature. Tab to calendar button is failing color contras for focus visible.
## Expected results:
Color contrast of keyboard visible focus should have a minimum of 3:1 ratio when it goes to any tools in toolbar section.

## User Impact:
Low vision Users might fail to acknowledge the focused element if it doesn't have minimum color contrast ratio.
## Attachments:
![Untitled](https://user-images.githubusercontent.com/100552367/207831764-0bc9ada7-bb8e-4012-adb7-095a6273e18d.png)

**Comment by Jtith** [comment]
verified in the latest environment, issue is fixed, hence, closing the bug.

![image](https://user-images.githubusercontent.com/100552367/211790593-e582a834-4153-43e5-a7fc-c1c79ead0dad.png)

**Comment by fsteffi** [comment]
GitHubTags:#Closed;
